packages/core/src/internal/utils/supports.spec.ts

```diff
@@ -1,16 +1,11 @@
 /*
- * Copyright (c) 2016-2020 VMware, Inc. All Rights Reserved.
+ * Copyright (c) 2016-2021 VMware, Inc. All Rights Reserved.
  * This software is released under MIT license.
  * The full license information can be found in LICENSE in the root directory of this project.
  */
 
-import { supportsFlexGap, supportsResizeObserver } from './supports.js';
-
 describe('browser support feature checks', () => {
   it('should check if browser supports ResizeObserver', () => {
-    expect(supportsResizeObserver()).toBe(true);
-  });
-  it('should check if browser supports FlexGap', () => {
-    expect(supportsFlexGap()).toBe(true);
+    expect(document.body.getAttribute('cds-supports')).toBe('js flex-gap');
```

Comment:
It seems like this test is more brittle than it needs to be. Can we test if the attribute contains those two strings? Instead of exact matching them in that order?

Reply:
Split into two assertions